config/clb-consumer/contact-data/delta-sync-dispatching.js

```diff
@@ -0,0 +1,123 @@
+const {
+  deleteFromPublicGraph,
+  insertIntoPublicGraph,
+  deleteFromSpecificGraphs,
+  insertIntoSpecificGraphs
+} = require('./util');
+
+const {
+  LANDING_ZONE_GRAPH
+} = require('./config')
+/**
+* Dispatch the fetched information to a target graph.
+* Note: <share://file/data> will be ADDED to it's own graph.
+*   We take only care of adding them, not updating triples, this is a TODO
+* @param { mu, muAuthSudo } lib - The provided libraries from the host service.
+* @param { termObjectChangeSets: { deletes, inserts } } data - The fetched changes sets, which objects of serialized Terms
+*          [ {
+*              graph: "<http://foo>",
+*              subject: "<http://bar>",
+*              predicate: "<http://baz>",
+*              object: "<http://boom>^^<http://datatype>"
+*            }
+*         ]
+* @return {void} Nothing
+*/
+
+const publicTypes = [
+  '<http://lblod.data.gift/vocabularies/organisatie/TypeVestiging>'
+]
```

Comment:
**Nitpick:** Please do not add code in between a function and its documentation.